### PR TITLE
Makefile Ebook CTA

### PR DESCRIPTION
**Pull request by adamgordonbell**
Makefile articles now have CTA at the end of the article pointing to [here](https://newsletter.earthly.dev/makefile)

We can play around with the wording if people are interested but for now the bottom of each article says this:

If you enjoyed this tutorial, you might want to grab a copy of the free eBook [“Makefile ins and outs”](https://newsletter.earthly.dev/makefile). In it, you’ll learn many tips and tricks for structuring builds and scaling up a build process. From how to write a simple Makefile to how to deal with dependencies and everything in between. Grab it [right now](https://newsletter.earthly.dev/makefile)!

### CODE REVIEW

**Comment by thtmnisamnstr** [comment]
I think the copy and links are fine. Looking at it in the preview below, do you think we should italicize the CTA?
image

**Comment by adamgordonbell** [comment]
>  Looking at it in the preview below, do you think we should italicize the CTA?
I can try that if you think it would look better. I've updated it to use growthbook. Could you verify I've done it right @thtmnisamnstr ? 

https://app.growthbook.io/features/blog-bottom-cta-20230501

File: blog/_includes/growthbook.html [hunk 1]
```diff
@@ -47,7 +47,19 @@
...
             break;
         }
-      }*/
+      }
+
+      if(document.getElementById("makefile-cta-control")){
+        if(gb.isOn("makefile-blog-bottom-cta-20230615")) {
+            console.log("makefile-blog-bottom-cta-20230615 on");
+            document.getElementById("makefile-cta-control").style.display = "none";
+            document.getElementById("makefile-cta-change").style.display = "";
```
Comment (thtmnisamnstr) [inline blog/_includes/growthbook.html:56]: @adamgordonbell I think this should be `...style.display = "block" (or "inline" or whatever the element should have)

File: blog/_includes/growthbook.html [hunk 2]
```diff
@@ -47,7 +47,19 @@
...
+          } else {
+            console.log("bmakefile-blog-bottom-cta-20230615 off");
+            document.getElementById("makefile-cta-control").style.display = "";
```
Comment (thtmnisamnstr) [inline blog/_includes/growthbook.html:59]: Same here. I think this should be `...style.display = "block" (or "inline" or whatever the element should have).

**Review by thtmnisamnstr — APPROVED** [review]

**Pull request merged**